Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -208,28 +202,21 @@ class CustomerIOImplementation: CustomerIOInstance {
      is no active customer, this will fail to register the device
      */
     public func registerDeviceToken(_ deviceToken: String) {
-        addDeviceAttributes(deviceToken: deviceToken)
-    }
+        logger.info("initiating device token registration for \(deviceToken)")
```

Comment:
I deliberately renamed this log message as customers using our tracking module would have seen same log twice, one from CustomerIOImplementation, one from DataPipeline

Reply:
Removed this as we decided to remove more checks